tests/Net/URL2Test.php

```diff
@@ -762,6 +767,8 @@ public function test19315()
             Net_URL2::removeDotSegments($nonStringObject);
         } catch (PHPUnit_Framework_Error $error) {
             $this->addToAssertionCount(1);
+        } catch (Throwable $error) {
```

Comment:
Throwable is only in PHP7/PHP 8... So this presuambly isn't going to work on some of the old old versions of PHP we're still in theory trying to test on (PHP 5.3)?

https://www.php.net/manual/en/class.throwable.php

I did query which version of PHP we should be supporting (and such, testing) in https://pear.php.net/bugs/bug.php?id=28662

Reply:
That's not a problem, it's forwards compatible. The tests show that as well and cover PHP 5.3...8.4.